From c33c275a9569981655095e8a2f9475d97cc762c6 Mon Sep 17 00:00:00 2001
From: wouterlucas <wouter@wouterlucas.com>
Date: Tue, 23 May 2017 20:50:09 -0700
Subject: [PATCH 56/97] Rely on partner bridge override instead of setting up
 entire bridge with all resources. Use the default PartnerBridge in
 src/platform/gibbon/resources/default/ unless NETFLIX_USE_KEYMAP value is
 specified. Currently only added to the posix platform.

---
 .../{bcm/PartnerBridge.js => posix/PartnerBridge_BCM.js}     | 0
 partner/platform/posix/platform.cmake                        | 5 +++++
 2 files changed, 5 insertions(+)
 rename partner/platform/{bcm/PartnerBridge.js => posix/PartnerBridge_BCM.js} (100%)

diff --git a/partner/platform/bcm/PartnerBridge.js b/partner/platform/posix/PartnerBridge_BCM.js
similarity index 100%
rename from partner/platform/bcm/PartnerBridge.js
rename to partner/platform/posix/PartnerBridge_BCM.js
diff --git a/partner/platform/posix/platform.cmake b/partner/platform/posix/platform.cmake
index f047ba1b..664dc18c 100644
--- a/partner/platform/posix/platform.cmake
+++ b/partner/platform/posix/platform.cmake
@@ -11,3 +11,8 @@ gibbon_set_eventloop("posix")
 gibbon_add_js(partner_signup.js)
 gibbon_add_js(partner_sso.js)
 
+if(NETFLIX_USE_KEYMAP)
+	message("  - Installing ${NETFLIX_USE_KEYMAP} as PartnerBridge.js")
+	file(RENAME ${CURRENT_DIRECTORY}/${NETFLIX_USE_KEYMAP} ${CURRENT_DIRECTORY}/PartnerBridge.js)
+endif()
+
-- 
2.17.1

